man/portablectl.xml

```diff
@@ -352,6 +352,13 @@
         <listitem><para>Don't block waiting for attach --now to complete.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--extension=</option><replaceable>PATH</replaceable></term>
+
+        <listitem><para>Attach an additional image on top of <replaceable>IMAGE</replaceable>, using an
+        overlay-like setup.</para></listitem>
```

Comment:
"overlay-like" is unclear... Maybe `Attach an additional image <replaceable>PATH</replaceable> as an overlay on top of <replaceable>IMAGE</replaceable>`?

Reply:
should also say that this may be used multiple times on the same command line to attach multiple images. and say in which order the stuff is applied then

Reply:
changed - it's IMAGE only for now, pid1 will need ExtensionDirectory= for dirs too (on the TODO list together with nspawn support)

Reply:
> should also say that this may be used multiple times on the same command line to attach multiple images. 

actually at the moment multiple --extension results in services being attached separately - I'll fix that (when I implemented it last year I was thinking about future improvement to join namespaces of separate portables)

Reply:
done:

```
$ cat /run/systemd/system.attached/app0.service.d/20-portable.conf
# Drop-in created for image '/home/bluca/git/systemd/app0.raw', do not edit.

[Service]
RootImage=/home/bluca/git/systemd/image.raw
Environment=PORTABLE=app0.raw
BindReadOnlyPaths=/etc/os-release:/run/host/os-release
LogExtraFields=PORTABLE=app0.raw
ExtensionImages=/home/bluca/git/systemd/app0.raw
ExtensionImages=/home/bluca/git/systemd/app1.raw
```